### PR TITLE
Added sorting for events

### LINKED ISSUE #8046: Sort log update events by taken at time rather than event created at time
### Describe the issue

When users are entering historical log updates into the system, they are often confused by the event log. The confusion arises because events are currently sorted by the date and time they were triggered, rather than by the "measured at" timestamp entered by the users during the log update. This makes it difficult for users to distinguish between past data and the latest changes.

### To Replicate
1. File a log update with the `measured at` date as today
2. File a log update with the `measured at` date as yesterday or any past date
3. See the events, it's currently sorted based on event creation date, which might be unintuitive for a normal user.

### Screenshots
image

### Suggested Solution

- sort the events log date based on the "measured at" value 
- the field that doesn't have "measured at " value, sort them based on "created at" value

cc: @nihal467 @sainak @gigincg @aparnacoronasafe 

Issue reported by athul doctor

### PR DESCRIPTION
**Pull request by khavinshankar**
- Fixes #8046
- Depends on https://github.com/coronasafe/care/pull/2324

### CODE REVIEW

**Review by rithviknishad — APPROVED** [review]

**Review by shivankacker — APPROVED** [review]

**Comment by nihal467** [comment]
waiting for the fix of https://github.com/coronasafe/care_fe/issues/8276

**Comment by nihal467** [comment]
image

- [x] make the sort by time taken more human-readable format

File: src/Locale/en/SortOptions.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
   "facility__name,-last_consultation__current_bed__bed__name": "Bed No. N-1",
   "-review_time": "Latest review date first",
   "review_time": "Oldest review date first",
+  "taken_at": "Latest taken date first",
+  "-taken_at": "Oldest taken date first",
```
Comment (khavinshankar) [inline src/Locale/en/SortOptions.json]:
```suggestion
  "taken_at": "Oldest taken date first",
  "-taken_at": "Latest taken date first",
```

**Comment by nihal467** [comment]
LGTM

**Pull request merged**